pkg/resources/resources.go

```diff
@@ -667,8 +685,7 @@ const (
 	PacketAPIKey    = "apiKey"
 	PacketProjectID = "projectID"
 
-	KubevirtKubeConfig    = "kubeConfig"
-	KubevirtCSIKubeConfig = "csiKubeConfig"
+	KubevirtKubeConfig = "kubeConfig"
```

Comment:
I know it was already there, but please rename this to `KubeVirtKubeconfig`

Reply:
done